scripts/Interface/ui/sans_isis/SANSSaveOtherWindow.py

```diff
@@ -29,6 +31,8 @@ def __init__(self, parent_widget=None):
         self.subscribers = []
         self.setup_view()
 
+        UsageService.registerFeatureUsage("Feature", "ISIS SANS - Save Other Tab", False)
```

Comment:
Why is this feature hyphen delimited?

Reply:
This is a leftover from before the discussion on how to layout the usage reporting of interfaces vs features. I've updated all of these